.travis.yml

```diff
@@ -0,0 +1,36 @@
+language: rust
+rust:
```

Comment:
What does this mean? Which version of Rust it is actually going to be using?

Reply:
Using travis rust language. according to travis doc, it will use current latest stable, beta, and nightly (This is the standard setup for an opensource crate) Also I added nightly-2019-05-22 (the version used by nearcore), so it run four builds with these four rust versions.

Reply:
These are several compiler versions, which one will it use? Will it try all of them?

Reply:
Will run all of them https://github.com/nearprotocol/borsh/runs/235219654